patches/extensions-browser-extension_event_histogram_value.h.patch

```diff
@@ -1,13 +1,14 @@
 diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
-index 8312660d4edae8639828f217e82db6a4b3d51a15..1f147c7d7fbb4e676750e793611c41ca997f2643 100644
+index 8312660d4edae8639828f217e82db6a4b3d51a15..ccb5ba3d84579da0c209fba927495dbd81d102fa 100644
 --- a/extensions/browser/extension_event_histogram_value.h
 +++ b/extensions/browser/extension_event_histogram_value.h
-@@ -439,6 +439,8 @@ enum HistogramValue {
+@@ -439,6 +439,9 @@ enum HistogramValue {
    ACCESSIBILITY_PRIVATE_ON_SELECT_TO_SPEAK_STATE_CHANGE_REQUESTED,
    INPUT_METHOD_PRIVATE_ON_FOCUS,
    SYSTEM_POWER_SOURCE_ONPOWERCHANGED,
 +  BRAVE_START = 5000,
 +  BRAVE_AD_BLOCKED,
++  BRAVE_SYNC_BROWSER_TO_BACKGROUND_PAGE,
```

Comment:
do we need histogram stats here?

Reply:
i think we don't need. I thought it is necessary. Will remove, thanks for pointing this.